src/sentry/static/sentry/app/views/eventsV2/utils.tsx

```diff
@@ -384,23 +384,31 @@ export function downloadAsCsv(tableData, columnOrder, filename) {
   link.remove();
 }
 
+type AggregateTransformer = (field: string) => string;
+
+// an array of 2-tuples of which the first tuple is an array of field names or aggregate names
+// to match a given column on, and the second tuple would transform the given column.
+const TRANSFORM_AGGREGATES: Array<[Array<string>, AggregateTransformer]> = [
+  [['p99', 'p95', 'p75'], () => 'transaction.duration'],
```

Comment:
What about apdex and impact?

Reply:
`apdex` and `impact` will now be dropped.